Traceback (most recent call last):
  File "main.py", line 290, in <module>
  File "main.py", line 25, in __init__
  File "main.py", line 31, in root
  File "main.py", line 51, in fill_combobox
  File "WriteAndReadFilesFunctions.py", line 24, in planet_classes
sqlite3.OperationalError: unable to open database file
